### PR TITLE
Migration mediainfo to ES6 modules 

### PR DESCRIPTION
**Pull request by grafixeyehero**
**Issues**
Part of #868

### CODE REVIEW

File: src/components/mediainfo/mediainfo.js [hunk 1]
```diff
@@ -48,68 +57,68 @@ define(['datetime', 'globalize', 'appRouter', 'itemHelper', 'indicators', 'mater
...
                 if (item.EndDate) {
                     date = datetime.parseISO8601Date(item.EndDate);
-                    text += ' - ' + datetime.getDisplayTime(date);
+                    text += ` - ${datetime.getDisplayTime(date)}`;
                 }
 
                 miscInfo.push(text);
             } catch (e) {
-                console.error('error parsing date: ' + item.StartDate);
+                console.error(`error parsing date: ${item.StartDate}`);
```
Comment (dkanada) [inline src/components/mediainfo/mediainfo.js:69]:
```suggestion
                console.error('error parsing date:', item.StartDate');
```

File: src/components/mediainfo/mediainfo.js [hunk 2]
```diff
@@ -143,7 +152,7 @@ define(['datetime', 'globalize', 'appRouter', 'itemHelper', 'indicators', 'mater
                     text = datetime.toLocaleDateString(date);
                     miscInfo.push(text);
                 } catch (e) {
-                    console.error('error parsing date: ' + item.PremiereDate);
+                    console.error(`error parsing date: ${item.PremiereDate}`);
```
Comment (dkanada) [inline src/components/mediainfo/mediainfo.js:155]: same

File: src/components/mediainfo/mediainfo.js [hunk 3]
```diff
@@ -176,7 +185,7 @@ define(['datetime', 'globalize', 'appRouter', 'itemHelper', 'indicators', 'mater
                     miscInfo.push(text);
                 }
             } catch (e) {
-                console.error('error parsing date: ' + item.StartDate);
+                console.error(`error parsing date: ${item.StartDate}`);
```
Comment (dkanada) [inline src/components/mediainfo/mediainfo.js:188]: same

File: src/components/mediainfo/mediainfo.js [hunk 4]
```diff
@@ -193,14 +202,14 @@ define(['datetime', 'globalize', 'appRouter', 'itemHelper', 'indicators', 'mater
...
+                        const endYear = datetime.parseISO8601Date(item.EndDate).getFullYear();
 
                         if (endYear !== item.ProductionYear) {
-                            text += '-' + datetime.parseISO8601Date(item.EndDate).getFullYear();
+                            text += `-${datetime.parseISO8601Date(item.EndDate).getFullYear()}`;
                         }
 
                     } catch (e) {
-                        console.error('error parsing date: ' + item.EndDate);
+                        console.error(`error parsing date: ${item.EndDate}`);
```
Comment (dkanada) [inline src/components/mediainfo/mediainfo.js:212]: Hmmm, I don't know. @MrTimscampi @ferferga maybe we want to start using literals for error messages?

File: src/components/mediainfo/mediainfo.js (hunk 1 above)
Comment (dmitrylyzo) [reply src/components/mediainfo/mediainfo.js:69]: why last `'` in `item.StartDate'`?

Comment (dkanada) [reply src/components/mediainfo/mediainfo.js:69]: fixed

File: src/components/mediainfo/mediainfo.js (hunk 4 above)
Comment (MrTimscampi) [reply src/components/mediainfo/mediainfo.js:212]: Seems good. We can also us `console.error('error parsing date:', item.EndDate);` and browsers should handle it.

Comment (dkanada) [reply src/components/mediainfo/mediainfo.js:212]: Do you think we should enforce one style over the other for most instances though? I think picking one would improve readability when parsing console lines in the code.

Comment (MrTimscampi) [reply src/components/mediainfo/mediainfo.js:212]: I think we should. Also make use of more advanced console utilities like groups, asserts, counts, traces, etc.

We can improve our readability and the usefulness of our logs a lot, I feel.

**Pull request merged**